lxd/instance/drivers/driver_common.go

```diff
@@ -1510,18 +1510,31 @@ func (d *common) devicesUpdate(inst instance.Instance, removeDevices deviceConfi
 				return nil, fmt.Errorf("Failed pre-start check for device %q: %w", dev.Name(), err)
 			}
 
-			_, err = dm.deviceStart(dev, instanceRunning)
+			runConf, err := dm.deviceStart(dev, instanceRunning)
 			if err != nil && err != device.ErrUnsupportedDevType {
 				return nil, fmt.Errorf("Failed to start device %q: %w", dev.Name(), err)
 			}
 
 			revert.Add(func() { _ = dm.deviceStop(dev, instanceRunning, "") })
 
-			devlxdEvents = append(devlxdEvents, map[string]any{
+			event := map[string]any{
 				"action": "added",
 				"name":   entry.Name,
 				"config": entry.Config,
-			})
+			}
+
+			if len(runConf.Mounts) > 0 {
+				for _, opt := range runConf.Mounts[0].Opts {
```

Comment:
This is fine for me regardless of my following comment, but thinking about making this more resilient to future changes, would it be better not to assume we are looking for only the first item of `runConf.Mounts`?

Reply:
The devlxd disk event can only take a single mount, so we can only take the first one anyway.